app/src/main/res/values-es/strings.xml

```diff
@@ -395,20 +397,20 @@
 
     <!-- Dax Dialog -->
     <string name="daxDialogHideButton">OCULTAR</string>
-    <string name="daxDialogPhew">¡Genial!</string>
+    <string name="daxDialogPhew">¡Uf!</string>
     <string name="daxDialogNext">Siguiente</string>
     <string name="daxDialogHighFive">¡Choca esos cinco!</string>
     <string name="daxDialogGotIt">Entendido</string>
     <string name="hideTipsTitle">¿Ocultar los consejos restantes?</string>
-    <string name="hideTipsText">Sólo hay unos cuantos, y tratamos de hacerlos informativos.</string>
+    <string name="hideTipsText">Solo hay unos cuantos, e intentamos que sean informativos.</string>
     <string name="hideTipsButton">Ocultar consejos para siempre</string>
 
     <!-- New Onboarding Experience -->
     <string name="onboardingWelcomeTitle">"¡Bienvenido a DuckDuckGo!"</string>
-    <string name="onboardingDaxText"><![CDATA[Internet puede ser un poco espeluznante.<br/><br/>¡No te preocupes! Buscar y navegar con privacidad es más fácil de lo que piensas.]]></string>
+    <string name="onboardingDaxText"><![CDATA[Internet puede ser un lugar horrible.\n\nNo te preocupes. Buscar y navegar de forma privada es más fácil de lo que piensas.]]></string>
```

Comment:
`<br/>` replaced by `\n` will not work here.

Reply:
done